src/kili/services/export/format/kili/common.py

```diff
@@ -84,9 +84,20 @@ def _save_assets_export(self, assets: List[Dict], output_filename: str):
         with TemporaryDirectory() as root_folder:
             base_folder = os.path.join(root_folder, self.project_id)
             os.makedirs(base_folder)
-            project_json = json.dumps(assets, sort_keys=True, indent=4)
-            with open(os.path.join(base_folder, "data.json"), "wb") as output_file:
-                output_file.write(project_json.encode("utf-8"))
+            if self.single_file:
+                project_json = json.dumps(assets, sort_keys=True, indent=4)
+                with open(os.path.join(base_folder, "data.json"), "wb") as output_file:
+                    output_file.write(project_json.encode("utf-8"))
+            else:
+                labels_folder = os.path.join(base_folder, "labels")
+                os.makedirs(labels_folder)
+                for asset in assets:
+                    external_id = asset["externalId"].replace(" ", "_")
+                    asset_json = json.dumps(asset, sort_keys=True, indent=4)
+                    with open(
+                        os.path.join(labels_folder, f"asset_{external_id}.json"), "wb"
```

Comment:
when importing labels on an asset called 'foo' from the CLI, we search for the file called 'foo.json' so maybe we should rather call the exported file 'foo.json' ?
Also the prefix asset is disturbing because it is not an asset but a label.

Reply:
👍